dpipe/predict/shape.py

```diff
@@ -113,6 +151,13 @@ def wrapper(x, *args, **kwargs):
 
             if stream:
```

Comment:
Throw `ValueError` for `stream == True and async_predict == True`

Reply:
Done